website/src/components/BrowserWindow/styles.module.css

```diff
@@ -77,5 +77,10 @@
 }
 
 .browserWindowBody {
+  background-color: var(--ifm-background-color);
   padding: 1rem;
 }
```

Comment:
Not a bug fix, but can we make the browser window have a bottom border radius like all other boxes?

Reply:
I wouldn't do that, often window of desktop software has rounded edges only at the top.

Reply:
This window looks sufficiently like a simulation for a macOS window, and macOS has round corners on all four edges. So if anything this would only be a more accurate simulation.

Reply:
Alight, however a bit unusual for me as an Ubuntu user, but if this window looks like on macOS, that change makes sense then.

Reply:
Thanks, I was wondering if there are Linux GUIs that look like that (without the border radius) 😄

Reply:
<img width="381" alt="image" src="https://user-images.githubusercontent.com/55398995/161971666-16d1fc19-ca65-411a-b4fa-bcbcaeb5d3c6.png">

Need to apply bottom border radius for the inner container